include/mapnik/svg/svg_path_attributes.hpp

```diff
@@ -34,6 +34,8 @@
 #include "agg_trans_affine.h"
 #pragma GCC diagnostic pop
 
+#include <tuple>
```

Comment:
minor thing: isn't this an implementation detail to be moved to the cpp file?

Reply:
There isn't a .cpp file, everything is defined in the header file.